creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -249,9 +249,10 @@ const _determineNewReplicaSetWhenOneNodeIsUnhealthy = (
   // If we already already checked this primary and it failed the health check, select the higher clock
   // value of the two secondaries as the new primary, leave the other as the first secondary, and select a new second secondary
   if (unhealthyReplicasSet.has(primary)) {
+    const secondary1Clock = replicaToUserInfoMap[secondary1]?.clock || -1
```

Comment:
@theoilie isn't this the old property?

Reply:
there are 2: when it's in monitor-state, find-sync-requests, or find-replica-set-updates then it's a mapping of all node->wallet->clock. when it's in update-replica-set then it's the filtered mapping of node->clock